Create Executive vote - July 11, 2024.md (#2690)
makerdao/community: governance/votes/Executive vote - July 11, 2024.md

---

> title: Template - [Executive Vote] Aave-SparkLend Revenue Share, Update Keeper Network Address, Stability Scope Parameter Changes, Delegate Compensation, Spark Proxy Spell - July 11, 2024

Author: This was a deliberate variation due to the change in the mechanism of payments.

summary: Transfer Dai to Aave as part of the SparkLend revenue sharing agreement, update the Chainlink Keeper Network Treasury Address, make Stability Fee and DSR changes as per the Stability Scope, and trigger a Spark Proxy Spell.
date: 2024-07-11T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Aave Revenue Share, Update Keeper Network Address, Stability Scope Parameter Changes, Delegate Compensation, Spark Proxy Spell - July 11, 2024

Author: "Aave Revenue Share" was consistent with the title of previous executives containing distributions of this nature

The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

> If this executive proposal passes, the following **changes** will occur within the Maker Protocol:

Author: Please suggest an update to the template if you are going to suggest variation from the template language

- **219,125 DAI** will be transferred to Aave as part of the Aave-SparkLend Revenue Sharing Agreement.
- The Chainlink Keeper Network Treasury Address will be updated.
- Multiple Stability Fees and the DSR will be updated.
- **209.17 MKR** will be distributed to qualified Aligned Delegates.
- A Spark Proxy Spell will be triggered at [0x91824fa4fd51E8440a122ffDd49C701F5C56D58e](https://etherscan.io/address/0x91824fa4fd51E8440a122ffDd49C701F5C56D58e).

> **Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Reviewer: Following the previous Executives and the comment above from VoteWizard, we should replace "changes and additions" with "actions".

> **Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Reviewer: ```suggestion
**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
```
Following the previous Executives and the comment above from VoteWizard, we should replace "changes and additions" with "actions".

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### Aave-SparkLend Revenue Share Payment

- **Authorization**: [Support Scope 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
- **Proposal**: [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-4-q2-2024/24572)

> If this executive proposal passes, a payment of **219,125 DAI** will be made to a smart contract controlled by Aave Governance at [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) as part of the Aave-SparkLend Revenue Share.

Reviewer: I'd suggest adding that the payment is corresponding to Q2 2024

Reviewer 2: Since it is detailed within the forum post, I'm going to leave this one out. Reasonable suggestion however, thanks for the careful review.

Author: Wording was deliberately chosen to be consistent with previous distributions of this nature.

e.g. https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20April%2022%2C%202024.md#aave-revenue-share-payment

### Update Chainlink Keeper Network Treasury Address

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/amend-keeper-network-chainlink-automation-v2-1/24593/3)
- **Proposal**: [Forum post](https://forum.makerdao.com/t/amend-keeper-network-chainlink-automation-v2-1/24593)

If this executive proposal passes, the Chainlink Keeper Network Treasury Address will be updated to [0xBE1cE564574377Acb17C2b7628E4F6dd38067a55](https://etherscan.io/address/0xBE1cE564574377Acb17C2b7628E4F6dd38067a55). This is a requirement to facilitate the upgrade of Chainlink Automation for MakerDAO from v2.0 to v2.1.

The required contract call is:

`keeperRegistry20.migrateUpkeeps([54617870736745152906494510799239970615562458165653206109311709851388548258865], keeperRegistry21);`

### Stability Scope Parameter Updates

- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594/2)
- **Proposal**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594)

> If this executive proposal passes, the following parameter changes will be made.

Reviewer: ":" at the end, rather than a period

> If this executive proposal passes, the following parameter changes will be made.

Reviewer: ```suggestion
If this executive proposal passes, the following parameter changes will be made:
```
":" at the end, rather than a period

#### [Stability Fee](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Reductions

- ETH-A: Decrease by 1 percentage point, from 8.25% to **7.25%**.
- ETH-B: Decrease by 1 percentage point, from 8.75% to **7.75%**.
- ETH-C: Decrease by 1 percentage point, from 8% to **7%**.
- WSTETH-A: Decrease by 1 percentage point, from 9.25% to **8.25%**.
- WSTETH-B: Decrease by 1 percentage point, from 9% to **8%**.
- WBTC-A: Decrease by 1 percentage point, from 9.75% to **8.75%**.
- WBTC-B: Decrease by 1 percentage point, from 10.25% to **9.25%**.
- WBTC-C: Decrease by 1 percentage point, from 9.5% to **8.5%**.

#### [Dai Savings Rate (DSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-the-dai-savings-rate) Reduction

The DSR will decrease by 1 percentage point, from 8% to **7%**.

> ### Delegate Compensation

Reviewer 3: ```suggestion
### Aligned Delegate Compensation for June 2024
```

**Authorization**: [Atlas 2.6 Aligned Delegates](https://mips.makerdao.com/mips/details/MIP101#2-6-aligned-delegates-ads-gov6)  
**Proposal**: [Forum post](https://forum.makerdao.com/t/june-2024-aligned-delegate-compensation/24617)

> If this executive proposal passes a total of **209.17 MKR** will be distributed for Aligned Delegate Compensation as follows. This is the first month where Aligned Delegates may request payments to multiple addresses. If a payout is sent to a different recipient at an Aligned Delegate's request, the Aligned Delegate's name will appear in parentheses for transparency. 

Reviewer: ```suggestion
If this executive proposal passes a total of **209.17 MKR** will be distributed for Aligned Delegate Compensation as detailed below.
```
I'd suggest to change the language because it's usually expected that the detail follows directly after the "as follows" statement.

| WBC         | 1.34        |[0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |

### Spark Proxy Spell

**Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594/2), [Governance Poll 1120](  https://vote.makerdao.com/polling/QmTBsxR5)  
**Proposal**: [Stability Scope Parameter Proposal](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594), [[Jun 27, 2024] Proposed Changes to Spark for Upcoming Spell](https://forum.makerdao.com/t/jun-27-2024-proposed-changes-to-spark-for-upcoming-spell/24552)

If this executive proposal passes a Spark Proxy spell will be triggered at [0x91824fa4fd51E8440a122ffDd49C701F5C56D58e](https://etherscan.io/address/0x91824fa4fd51E8440a122ffDd49C701F5C56D58e). The expected inclusions are as follows:

#### [Mainnet] Spark Effective DAI Borrow Rate Reduction

The Spark Effective DAI Borrow Rate will be reduced by 1 percentage point, from 9% to **8%**.

#### [Mainnet] Increase Capacity of weETH

> - Increase weETH Supply Cap Max by 150,000 weETH, from 50,000 weETH to **200,000 weETH**

Reviewer: ```suggestion
- weETH Supply Cap Max will be increased by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
```
for the sake of consistency in style and form

Reviewer 2: This might be my error from how I inputted the items into the Exec sheet. To keep this as consistent as possible with the sheet language, I'm going to change the DAI borrow rate item. That way the form is the same across these three, and spell sheet is still largely word for word with what is being communicated to voters. 

Do we have a section about this for the style guide? Because while the sheet should say "Increase to X" I do like the "will be increased" format we've been using for the spell copy. Maybe the convention should be that the sheet describes the change and the format for the copy becomes "[Parameter] will be [increased/reduced]..."

Author: It was taken verbatim from the poll: https://github.com/makerdao/community/blob/master/governance/polls/SparkLend%20Ethereum%20-%20Increase%20Capacity%20of%20weETH%20-%20July%201%2C%202024.md#review

> - Increase weETH [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features) Debt Ceiling by 150 million DAI, from 50 million DAI to **200 million DAI**

Reviewer: ```suggestion
- weETH [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features) Debt Ceiling will be increased by 150 million DAI, from 50 million DAI to **200 million DAI**
```
same as above

Reviewer 2: See above comment.

Author: If we are describing these as "actions" then the action is to "increase X" not "X will be increased"
